docs/developers/add-new-bidder.md

```diff
@@ -33,6 +33,13 @@ Bidder implementations are scattered throughout several files.
 
 Bidder implementations may assume that any params have already been validated against the defined json-schema.
 
+###Long form video support
+If bidder is going to support long form video make sure bidder returns:
+
+- single [IAB subcategory](https://iabtechlab.com/standards/content-taxonomy/) for each bid in `TypedBid.bid.Cat` field
```

Comment:
Will this `TypeBid.bid.Cat` be a string representing one of the `Content Taxonomy v2 with Tiered Categories` found in the excel file downloadable from `https://iabtechlab.com/standards/content-taxonomy/`? Or will this field be an `int` representing the `UniqueID` found in said excel file? Maybe it is an object with both the `UniqueID` and `Tier1...4` categories. Can we be a little more specific?

Reply:
@guscarreon link updated, please review again

Reply:
@guscarreon thank you, I added a little more info, please review again. Sorry about this.